templates/zerver/help/include/change-password-via-email-confirmation.md

```diff
@@ -0,0 +1,13 @@
+{start_tabs}
+
+1. If you are logged in, start by [logging out](/help/logging-out).
+
+2. Go to your organization's login page at `https://<organization-url>/login/`.
+
+3. Below the **Password** field, click **Forgot password**.
```

Comment:
This should be:
```suggestion
3. Below the **Password** field, click **Forgot your password?**.
```
Also, the link is not right below the Password field, so maybe we should say something like "below the authentication providers"?
<img width="345" alt="image" src="https://user-images.githubusercontent.com/2343554/197900677-0124eee0-7c0b-4c30-819b-6125b20147d0.png">